test/e2e/suite/certificates/additionaloutputformats.go

```diff
@@ -73,7 +74,7 @@ var _ = framework.CertManagerDescribe("Certificate AdditionalCertificateOutputFo
 		crt, err = f.Helper().WaitForCertificateReadyAndDoneIssuing(crt, time.Minute*2)
 		Expect(err).NotTo(HaveOccurred(), "failed to wait for Certificate to become Ready")
 
-		return crt
+		return crt.Name, crt
```

Comment:
Why this change? 👀 callers can read the Name field already from `crt` (and could define their own crtName variable if they plan to overwrite crt themselves afterwards)

Reply:
You're right, we should of just returned the `crt.Name` here. The intention would be that we then force caller to do the retry loop to `get`+`update`..